### PR TITLE
Fix/form style

### PR DESCRIPTION
**Pull request by Amber-Song**
# Context

Link to [Notion ticket](https://www.notion.so/cloudcannon/Form-fixes-f5653be818dd47859d940f9b1646afdd?pvs=4)

# Additional information

I didn't do the "Comments should be added for form best practices" though.

# Testing (tester)

The reviewer should check on this branch:

- The code
- The component in Bookshop browser
- The site in CloudCannon (link here)

# Before merge (PR owner)

- [ ] Delete the test site in CloudCannon
- [ ] For "generic" components **only**: remove `- content_blocks` from `structures` in the YAML file (this was added for testing).

### CODE REVIEW

**Comment by AlyshaNolan** [comment]
The alignment of the groups was supposed to stay the same - just the comment for the input needed changing

**Comment by Amber-Song** [comment]
> The alignment of the groups was supposed to stay the same - just the comment for the input needed changing

Ah. Yes. Changed the comment and changed the alignment back.

**Review by adon-cloudcannon — CHANGES_REQUESTED** [review]
- Checkbox and radio group ‘alignment’ text needs changing - copied from labelled icon and not changed to this context ✅
- If no label is provided for a checkbox/radio group, the little space in the border should be removed (i.e. don’t render an empty element) ✅
- Comments should be added for form best practices - e.g. “You should provide a label for groups” or “You should have at least one radio option checked by default”… something like that? ✅
- Textarea input should have a min-height to stop user from shrinking it too small ✅
- **Bug: radio group - clicking labels doesn’t select the radio input** - this still appears to not work? Just adding a radio button group and adding two buttons, the IDs appear to be the same? Request another look please :)

**Comment by Amber-Song** [comment]
> * **Bug: radio group - clicking labels doesn’t select the radio input** - this still appears to not work? Just adding a radio button group and adding two buttons, the IDs appear to be the same? Request another look please :)

Good spotting. It is fixed. This is because in ratio button group component radio buttons list, we didn't use bookshop component.

**Comment by AlyshaNolan** [comment]
It probably shouldn't use the subcomponent but should pass through a UUID for each input

I think the subcomponent shouldn't be used because of the group_name and required inputs

File: component-library/components/generic/form/radio-button-group/radio-button-group.bookshop.yml [hunk 1]
```diff
@@ -12,9 +12,7 @@ blueprint:
   label: ""
   id:
   required: false
-  radio_buttons: 
-    - label:
-      checked: false
+  radio_buttons: [bookshop:generic/form/radio-input]
```
Comment (AlyshaNolan) [inline component-library/components/generic/form/radio-button-group/radio-button-group.bookshop.yml]:
```suggestion
  radio_buttons:
    - label:
      checked:
      input_id:
```

**Comment by Amber-Song** [comment]
@adon-cloudcannon @AlyshaNolan It works now.

**Comment by AlyshaNolan** [comment]
There is a typo in the input (ratio_id -> radio_id) and then this looks great other than removing the content blocks from the page 😄

**Review by AlyshaNolan — APPROVED** [review]

**Pull request merged**